aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp

```diff
@@ -385,10 +386,65 @@ torch::class_<LinearPackedParamsBase> register_linear_params() {
   return register_linear_params;
 }
 
+torch::class_<EmbeddingPackedParamsBase> register_embedding_params() {
+  // Type for __getstate__/__setstate__ serialization
+  //
+  // Element 0 is a string to say what kind of EmbeddingParam this is
+  // Element 1 is the Tensors contained in the Param instance
+  // Element 2 is the double values (if any) contained in the Param instance
+  // Element 3 is the int values (if any) contained in the Param instance
+
+  using EmbeddingParamsSerializationType = std::tuple<
+    std::string,
+    std::vector<at::Tensor>,
+    std::vector<double>,
+    std::vector<int64_t>>;
+
+  static auto register_embedding_params =
+    torch::class_<EmbeddingPackedParamsBase>(
+      "quantized", "EmbeddingPackedParamsBase")
+      .def_pickle(
+          [](const c10::intrusive_ptr<EmbeddingPackedParamsBase>& params)
+              -> EmbeddingParamsSerializationType { // __getstate__ call
+            at::Tensor weight = params->unpack();
+            std::vector<at::Tensor> tensors_to_serialize = {weight};
+            std::vector<double> doubles_to_serialize = {};
+            int bit_rate = params->bit_rate();
+            int version = params->version();
+            std::vector<int64_t> longs_to_serialize = {bit_rate, version};
```

Comment:
hmm, should we have a separate field for version(probably as a first field as well)? in case the data format changes depending on version, we can have an easy access to version and decide what to do next

Reply:
Sounds good.